consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseConsentAPI.java

```diff
@@ -485,4 +479,28 @@ public Election retrieveElection(Integer electionId, String consentId) {
         }
         return election;
     }
+
+    @Override
+    public List<ConsentGroupNameDTO> verifyConsentsGroupNames(List<ConsentGroupNameDTO> consentGroupNames) {
```

Comment:
i would change the name of this method to indicate it will actually do the update if the names are valid. something like verifyAndUpdateConsentGroupNames or updateConsentGroupNamesWhenValid